sync/clients/sync.py

```diff
@@ -194,6 +235,32 @@ def onboard_workflow(self, workspace_id, job_id: str, project_id: str) -> dict:
             )
         )
 
+    def get_latest_project_config_recommendation(self, project_id: str) -> ProjectConfiguration:
```

Comment:
Are you sure this returns `ProjectConfiguration`, it looks like its returning a `dict` below

Reply:
Good catch. I'll think of a better way to handle the case if there is no Recommendation provided